core/modules/book/book.module

```diff
@@ -334,8 +334,10 @@ function book_block_configure($delta = '', $settings = array()) {
     '#title' => t('Book navigation block display'),
     '#options' => $options,
     '#default_value' => $settings['book_mode'],
-    '#description' => t("If <em>Show block on all pages</em> is selected, the block will contain the automatically generated menus for all of the site's books. If <em>Show block only on book pages</em> is selected, the block will contain only the one menu corresponding to the current page's book. In this case, if the current page is not in a book, no block will be displayed. The <em>Page specific visibility settings</em> or other visibility settings can be used in addition to selectively display this block."),
+    '#description' => t('Add additional visibility conditions to display this block more selectively.'),
   );
+  $form['book_mode']['all pages']['#description'] = t('The block will contain the automatically generated menus for all of the site\'s books.');
+  $form['book_mode']['book pages']['#description'] = t('The block will contain only the one menu corresponding to the current page\'s book. If the current page is not in a book, no block will be displayed.');
```

Comment:
This is a nice use case where we'd use double quotes, in order to avoid having to escape the apostrophes:

```suggestion
  $form['book_mode']['all pages']['#description'] = t("The block will contain the automatically generated menus for all of the site's books.");
  $form['book_mode']['book pages']['#description'] = t("The block will contain only the one menu corresponding to the current page's book. If the current page is not in a book, no block will be displayed.");
```

Reply:
Double quotes is slower in PHP because PHP is searching for variable replacements. Since there are no variables to replace in the strings here, I think escaping is the better option.

Reply:
That's not what our coding standards say though: https://docs.backdropcms.org/php-standards#quotes